README.md

```diff
@@ -242,18 +242,61 @@ Also missing are the following.
 
 ## Installing the reference implementation
 
+Currently, there are two ways of installing the reference implementation.
+You may install it on an existing Raspbian system or create a new image.
+The two alternatives are presented below.
+
+### Install on an existing system
+
 You'll need to install the required libraries and project.
 Here is how you can do it under the Ubuntu GNU/Linux distribution.
 
 ```bash
 sudo apt-get install git libbluetooth-dev libglib2.0-dev python3-dev python3-setuptools shellcheck sqlite3 virtualenv dh-virtualenv debhelper supervisor
-git clone https://github.com/dspinellis/epidose
+git clone https://github.com/eellak/epidose
 cd epidose
 virtualenv venv -p /usr/bin/python3
 . venv/bin/activate
 pip3 install -e .
 ```
 
+### Create a new image
+
+You'll need to download a vanilla Raspbian distribution (Lite version is suggested for this purpose)
+and burn it on a MicroSD card.
+Note that you must enable the SSH daemon on the corresponding image.
+Then execute the following steps:
+
+* In the /etc/ansible/ansible.cfg file uncomment and set as True the following:
+```bash
+allow_world_readable_tmpfiles = True
+pipelining = True
+```
+
+* In the /etc/ansible/hosts file create the group of usernames and  IP addresses for each Raspberry device, e.g.,
+```bash
+   [epidose]
+   192.168.0.105 ansible_user=pi ansible_connection=ssh ansible_private_key_file=~/ssh/id_rsa
+```
+
+* Execute the following commands (see explanation below about the ansible command-line arguments):
+```bash
+sudo apt-get install git ansible
+git clone https://github.com/eellak/epidose
+cd epidose
+sudo ansible-playbook install_and_configure_device.yml --tags "setup"
+sudo ansible-playbook install_and_configure_device.yml --tags "production|development" --extra-vars "epidose_network_psk=something eduroam_network_psk=something cuckoo_distribution_server_psk=something"
+```
+_tags_
+* setup enables a passwordless ssh for all the nodes username and hostname added inside the /etc/ansible/hosts
```

Comment:
Please describe: what is the recommended way to use these tags?

Reply:
I guess like, for the setup we recommend of using this tag first without any extra-vars?

Reply:
It's up to you to describe the correct installation process for the two cases you identified (and for the three tags).  The end result should be that somebody reading only README should be able to create a development environment and, more importantly, a working device.

Reply:
Ok, I am discussing it with Konstantinos Kravvaritis.
I will ask him to execute the steps and document the points that he struggles.
Afterwards, I will apply the necessary changes.

Reply:
Excellent!  Try to communicate only through `README` and script revisions.

Reply:
Ok!